src/libmongoc/tests/TestSuite.c

```diff
@@ -1123,9 +1114,7 @@ TestSuite_Destroy (TestSuite *suite)
       fclose (suite->outfile);
    }
 
-   if (suite->mock_server_log_buf) {
-      mcommon_string_free (suite->mock_server_log_buf, true);
-   }
+   mcommon_string_destroy (suite->mock_server_log_buf);
```

Comment:
```suggestion
   if (suite->mock_server_log_buf) {
      mcommon_string_append_destination_destroy (suite->mock_server_log_buf);
      bson_free (suite->mock_server_log_buf);
   }
```
See suggestion above regarding `TestSuite::mock_server_log_buf`.